app/views/companies/_companies.html.erb

```diff
@@ -3,22 +3,25 @@
       <tr>
         <th scope="col">#</th>
         <th scope="col">Joint Stock Company</th>
-        <th scope="col">Symbol</th>
+        <th scope="col">Symbol/ISIN</th>
         <th scope="col">Stock</th>
         <th scope="col">Shares amount</th>
         <th scope="col">Free float</th>
       </tr>
     </thead>
     <tbody>
+      <% index = 0 %>
```

Comment:
You should be able to use `each_with_index` instead of this :)

Reply:
The problem here is that not every iteration is displayed, and then index will be like 1,2,3,12,13,14,524 for example.
I like to have here counted every object, but only displayed objects.
I mean that in the displayed table should be numerated objects.